README.md

```diff
@@ -7,3 +7,140 @@ Requirements
 ------------
 
 - [Django](https://docs.djangoproject.com) 2.2.x
+- [Django AppSettings](https://pypi.org/project/django-app-settings/)
+- [lxml](https://pypi.org/project/lxml/)
+- [xmlsec](https://pypi.org/project/xmlsec/)
+- [Apache Ignite Python client](https://pypi.org/project/pyignite/) (optional)
+
+Specific Proxy Service
+----------------------
+
+### Usage
+
+To use eIDAS Proxy Service, adjust Django settings:
+
+* Set up  Django template engine and static file.
+* Add `'eidas_node.proxy_service.apps.ProxyServiceConfig'` to `INSTALLED_APPS`.
+* Set `ROOT_URLCONF` to `'eidas_node.proxy_service.urls'` or include it in your URL configuration.
+* Provide mandatory configuration options.
+
+```py
```

Comment:
I would rather see the example settings somewhere in a separate file.

Reply:
Moved to `samples/proxy_service_settings.py`.